components/layout/header.vue

```diff
@@ -9,7 +9,8 @@
             <BrandLogo class="w-24 h-24" />
           </NuxtLink>
         </div>
-        <div class="-mr-2 -my-2 md:hidden">
+        <div class="font-medium text-gray-800">Hotline 24/7</div>
```

Comment:
According to the prototype layout of the slide over panel of the menu is longer  - takes the whole side of the screen in mobile mode

Reply:
Okay I should see that in mobile phone for testing